packages/x-date-pickers/src/DateCalendar/PickersFadeTransitionGroup.tsx

```diff
@@ -50,7 +54,10 @@ export function PickersFadeTransitionGroup(inProps: PickersFadeTransitionGroupPr
   }
 
   return (
-    <PickersFadeTransitionGroupRoot className={clsx(classes.root, className)}>
+    <PickersFadeTransitionGroupRoot
+      className={clsx(classes.root, className)}
+      ownerState={{ ...other }}
```

Comment:
I did not want the `ownerState` receiving the `children` prop, it seemed counter intuitive. 🙈

Reply:
Do you mean I'm passing `ownerState` at all?
Or why not store it in a variable and pass it?
P.S. Any `styled` component needs `ownerState` to adhere to the exposed `styleOverrides` contract.

Reply:
I have moved the `ownerState` declaration outside of JSX.

Reply:
No my question was why `ownerState={{ ...other }}` instead of `ownerState={other}`

Reply:
Whoops, sorry for my "broken brain" that day. 🙈 
I will address it with the next PR. 😉